sdk/Lusid.Sdk.Tests/tutorials/Instruments/ExoticExamples.cs

```diff
@@ -64,5 +64,12 @@ public void ExoticCreationAndUpsertionExample()
             Assert.That(roundTripExotic.Content, Is.EqualTo(exotic.Content));
             Assert.That(roundTripExotic.InstrumentFormat, Is.EqualTo(exotic.InstrumentFormat)); 
         }
+        
+        [TestCase(ModelSelection.ModelEnum.SimpleStatic)]
+        public void InterestRateSwapValuationExample(ModelSelection.ModelEnum model)
+        {
+            var exotic = InstrumentExamples.CreateExampleExotic();
```

Comment:
The function name implies valuing an interest rate swap, but you instantiate an exotic. Is this intentional? If so, add a comment explaining